package.json

```diff
@@ -17,7 +17,7 @@
     "lib"
   ],
   "engines": {
-    "node": "10.x - 16.x"
+    "node": "10.x - 18.x"
```

Comment:
This is where I'm not sure. Should this be 14.x-18.x if we only test for those versions?

I think we've previously said that that is a *breaking* release for us. so perhaps that should be reserved for a second, breaking PR.

Reply:
https://github.com/trussworks/react-uswds/issues/2129